test/fielddef.test.ts

```diff
@@ -22,6 +22,14 @@ describe('fieldDef', () => {
     it('should access flattened fields in expression', () => {
       expect(vgField({field: 'foo.bar\\.baz'}, {expr: 'datum'})).toEqual('datum["foo.bar.baz"]');
     });
+
+    it('should access argmin field in expression', () => {
+      expect(vgField({aggregate: {argmin: 'b'}, field: 'a'})).toEqual('argmin_b.a');
```

Comment:
toBe

Reply:
why toBe?

Reply:
Because it's just a string. `toBe` is a stricter comparison (like `===` instead of `==`) and stricter is better for tests.

Reply:
We better fix that in a separate PR.   (See all tests around this file.)